resources/lang/hu/faults.php

```diff
@@ -0,0 +1,20 @@
+<?php
+
+return [
+    "faults" => "Hibák",
+    "created_at" => "Időpont",
```

Comment:
létrehozva?

Reply:
I think "Időpont" isn't bad if you look at it in relation to the other column headers (Helyszín, Leírás, Állapot).
But if you think both creation and update dates should be displayed, I would also prefer "Létrehozva" and "Frissítve".

Reply:
OK, changed it.